docs/builders.md

```diff
@@ -98,9 +98,6 @@ spec:
     - name: sample-cluster-buildpack
       kind: ClusterBuildpack
       id: paketo-buildpacks/nodejs
-    - name: sample-cluster-store
-      kind: ClusterStore
-      id: paketo-buildpacks/nodejs
```

Comment:
maybe we could leave the `id` line to show that we can take buildpacks from the store if it is added to the builder spec

Reply:
Something like this?
```yaml
- id: paketo-buildpacks/nodejs
  version: 1.2.3
```